plugins/rich-editor/src/scripts/mountEditor.tsx

```diff
@@ -22,9 +22,9 @@ export default function mountEditor(containerSelector: string | Element) {
         throw new Error("Could not find the BodyBox to mount editor to.");
     }
 
-    const initialFormat = bodybox.getAttribute("format") || "Rich";
```

Comment:
Don't we still require coalescing default here?

Reply:
In person discussion summary:

We don't actually want it anymore. We'd rather treat no format as a plain text box than as a rich editor.